walkthroughs/week-5-tdd/project/src/main/webapp/script.js

```diff
@@ -24,14 +24,16 @@ function sendMeetingRequest() {
   // split it into an array of names
   const attendees = attendeesNamesString.split(/\s*,\s*/);
 
-// comma-separated list of names
-  const optionalAttendeesNamesString = document.getElementById('optional-attendees').value;
+  // comma-separated list of names
+  const optionalAttendeesNamesString =
+      document.getElementById('optional-attendees').value;
   // split it into an array of names
   const optionalAttendees = optionalAttendeesNamesString.split(/\s*,\s*/);
 
   // Create the request to send to the server using the data we collected from
   // the web form.
-  const meetingRequest = new MeetingRequest(duration, attendees, optionalAttendees);
+  const meetingRequest =
+      new MeetingRequest(duration, attendees, optionalAttendees);
```

Comment:
indentation again

Reply:
Ah clang-format-path does not run on js files, I just realised. Trying to find another tool that does now

Reply:
In fact this was clang-format, it does work on JS files. It was because the line length was too long.